addons/portal/static/src/scss/portal.scss

```diff
@@ -409,6 +409,14 @@ img, .media_iframe_video, .o_image {
 
 }
 
+.modal-dialog {
```

Comment:
Hmm, wouldn't it be better to remove the "border" class on that specific div.

And then, at the same spot as this commit:
https://github.com/odoo/odoo/pull/146159/commits/6d762876b3eeb1790334a82e71316f95f3b5f5ea

Do:

```scss
.o_portal_chatter_composer_body > div {
    border: 1px solid var(--o-border-color);
}
```

That way we have the same approach to apply both border changes?

Don't like the fact that currently they are not aligned, one is based on modal being on top and one is based on "o_portal_chatter" being on top.

Reply:
Hello @awa-odoo 

Changes are done,
Can you please review it again 

Thanks :blush: